File: 1546.py
numOfInput = int(input())
scoreArray = list(map(float, input().split()))            #list에 map해준다 float형으로 input값을 split해서
result = float(((sum(scoreArray)*100)/(numOfInput*max(scoreArray))))

print('%.2f' %result)    #계산된 result값을 소숫점 2째자리까지 포맷팅해서 출력

File: 1546.c
#include<stdio.h>

int main(){
    int numOfInput;
    int maxScore; 
    double editedTotal = 0;
    
    scanf("%d", &numOfInput);
    int scoreArray[numOfInput];
    
    for(int i = 0; i < numOfInput; i++){
        scanf("%d", &scoreArray[i]);    
    }
    
    for(int ithMaximum = 0; ithMaximum < numOfInput; ithMaximum++){
        for(int current = 0; current < numOfInput-1-ithMaximum; current++){
            if(scoreArray[current] > scoreArray[current+1]){
                int temp = scoreArray[current];
                scoreArray[current] = scoreArray[current+1];
                scoreArray[current+1] = temp;
            }
        }    
    }
    
    maxScore = scoreArray[numOfInput-1];
    
    for(int i = 0; i < numOfInput; i++){
        editedTotal = editedTotal + scoreArray[i];
    }
    
    double result = (editedTotal*100)/(maxScore*numOfInput);
    printf("%f", result);
        
    return 0;
}
